### DISCUSSION TITLE
https://johanneskonings.dev/blog/2023-09-08-aws-sso-steampipe/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#55] [Blog Post Comments]
# https://johanneskonings.dev/blog/2023-09-08-aws-sso-steampipe/

See how this setup can help to discover your AWS resources across all SSO accounts with a mix of Steampipe, docker, bash scripts, and AWS CLI

https://johanneskonings.dev/blog/2023-09-08-aws-sso-steampipe/

**Comment by JohannesKonings** [comment]
Updated: simplify scripts with [aws-sso-util](https://github.com/benkehoe/aws-sso-util)
